website/package.json

```diff
@@ -7,11 +7,12 @@
     "node": "12.x"
   },
   "dependencies": {
-    "@hashicorp/nextjs-scripts": "^13.2.0",
+    "@hashicorp/nextjs-scripts": "14.0.1",
     "@hashicorp/react-alert-banner": "4.2.0",
     "@hashicorp/react-button": "3.0.2",
     "@hashicorp/react-call-to-action": "0.2.8",
     "@hashicorp/react-case-study-slider": "3.0.0",
+    "@hashicorp/react-code-block": "3.0.3",
```

Comment:
Added this as an explicit dependency, since we do import styles in `pages/style.css`.